README.md

```diff
@@ -85,41 +86,43 @@ extension RouteType {
 At this point we can easily create an Alamofire Request:
 
 ```swift
-let request: Alamofire.Request =  GithubAPI.Repository.GetInfo(owner: "xmartlabs", repo: "Opera").request
+let request: Request =  GithubAPI.Repository.GetInfo(owner: "xmartlabs", repo: "Opera").request
 ```
 
-> Notice that `RouteType` conforms to `Alamofire.URLRequestConvertible` so having the manager we can create the associated `Alamofire.Request`.
+> Notice that `RouteType` conforms to `Alamofire.URLConvertible` so having the manager we can create the associated `Request`.
 
 We can also take advantage of the reactive helpers provided by Opera:
 
 ```swift
 request
-  .rx_collection()
-  .doOnNetworkError { (error: NetworkError) in
-    // do something when networking went wrong
-  }
-  .subscribeNext { (repositories: [Repository]) in
-    // do something when networking and Json parsing completes successfully
-  }
+  .rx.collection()
+  .subscribe(
+    onNext: { (repositories: [Repository]) in
+      // do something when networking and Json parsing completes successfully
+    },
+    onError: {(error: OperaError) in
```

Comment:
Yep, it should be error. That's the only reason the `doOnNetworkError` helper was created, because it casted the error to the desired type